### DISCUSSION TITLE
Custom voice input limit?

### DISCUSSION THREAD

**Discussion by chasebethay** [neonbjb/tortoise-tts#197] [General]
Hi,

I read somewhere (maybe reddit) that Tortoise only uses the first 6 seconds of audio. Is that true?

Additionally, how many audio files could you provide to improve the output? Is there a soft/hard limit where the bot will no longer gain any appreciable benefit? Or are as many clips as possible welcome?

Thanks,

**Comment by FurkanGozukara** [comment]
Interested in this as well

**Comment by neonbjb** [comment]
Yeah when I trained the model I only trained on 6 second reference clips. In retrospect I should have used longer clips. This can't be fixed without retraining the model.
